### PR TITLE
CLOUDP-106463: add rules, remove hooks

### PR DESCRIPTION
**Pull request by leo-ri**
changes for  [CLOUDP-104085](https://github.com/mongodb/mongodb-atlas-kubernetes/pull/346/files )

### CODE REVIEW

File: charts/atlas-cluster/README.md [hunk 1]
```diff
@@ -97,3 +97,27 @@ containers:
            name: my-project-atlas-cluster-admin-user
            key: connectionString.standardSrv
 ```
+
+## Notes:
```
Comment (rodrigovalin) [inline charts/atlas-cluster/README.md:101]: Please remove the colon at the end of the title; maybe change the title to "Upgrade Notes"

**Review by priyolahiri — APPROVED** [review]
LGTM

File: charts/atlas-cluster/README.md [hunk 2]
```diff
@@ -97,3 +97,27 @@ containers:
...
+
+Atlas-operator version 0.6.1+ has to delete finalizers - this change requires additional steps.
+
+Manually workaround for the update from Atlas-cluster-0.1.7:
```
Comment (rodrigovalin) [inline charts/atlas-cluster/README.md:105]:
```suggestion
Manual workaround for the update from Atlas-cluster-0.1.7:
```

File: charts/atlas-cluster/README.md [hunk 3]
```diff
@@ -97,3 +97,27 @@ containers:
...
+1) need to remove manually the "helm.sh/hook" from Atlasproject
```
Comment (rodrigovalin) [inline charts/atlas-cluster/README.md:106]: Please start the sentences with an upper case letter.

File: charts/atlas-cluster/README.md [hunk 4]
```diff
@@ -97,3 +97,27 @@ containers:
...
+
+```bash
+kubectl annotate atlasproject/opib6d6o6k helm.sh/hook-
```
Comment (rodrigovalin) [inline charts/atlas-cluster/README.md:109]: We should not use the name of the resource in this doc, this is different on each case;

We should use `--selector` to find the collection of objects to modify; we can use any labels that the `atlasproject` resource has.

**Review by rodrigovalin — COMMENTED** [review]
There are a few comments here and there. The only thing we need to change is the documentation on how to modify (annotate or remove annotations) on the resources. This needs to be done in a generic way in order for everyone to be able to _just run the commands_.

**Review by rodrigovalin — APPROVED** [review]
LGTM, thanks for the changes.

**Pull request merged**